Gemfile.lock

```diff
@@ -249,25 +249,25 @@ GEM
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
-    net-ssh (6.0.2)
+    net-ssh (6.1.0)
     netrc (0.11.0)
     nio4r (2.5.4)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
```

Comment:
### [nokogiri](https://nokogiri.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.10.9 → 1.10.10

[_[change-log](https://nokogiri.org/CHANGELOG.html), [source-code](https://github.com/sparklemotion/nokogiri)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/sparklemotion/nokogiri/compare/v1.10.9...v1.10.10).

These are the individual commits:
- (9091602) [ci: only manage the v1.10.x pipeline on this branch](https://github.com/sparklemotion/nokogiri/commit/90916022e38fd48c3a9e3763b0f47f834261d399)
- (c39e1b0) [Support fat binary gems for ruby-2.7](https://github.com/sparklemotion/nokogiri/commit/c39e1b082e73557207db1468e4b567727a384579)
- (f0c324c) [Merge branch '2029-windows-support-for-ruby-27-on-v110x' int…](https://github.com/sparklemotion/nokogiri/commit/f0c324c1d93a2880a32dc46cf8766ff132409806)
- (d2d3c18) [update CHANGELOG for v1.10.10](https://github.com/sparklemotion/nokogiri/commit/d2d3c18a6c2db5b448381573c1248fe480198003)
- (a9a3717) [version bump to v1.10.10](https://github.com/sparklemotion/nokogiri/commit/a9a3717154bdb99ed6d0d785736eb471c9d6a954)


</details>